client/asset/eth/eth_test.go

```diff
@@ -4254,6 +4255,7 @@ func testConfirmRedemption(t *testing.T, assetID uint32) {
 	}
 
 	for _, test := range tests {
+		fmt.Println(test.name)
```

Comment:
OK to leave, but how about some nice formatting. `fmt.Printf("###### %s ###### \n", test.name)`